src/ui/components/RegistrarList.tsx

```diff
@@ -58,7 +61,7 @@ type RegistrarItemProps = {
 
 function RegistrarItem({registrar, testID}: RegistrarItemProps) {
   const {account, id, formattedFee} = registrar;
-
+  const navigation = useNavigation<NavigationProp<AppStackParamList>>();
```

Comment:
Same here, do not use the hook in every item. Instead it can be moved to parent and every item can receive a callback.